alphastats/DataSet_Preprocess.py

```diff
@@ -334,6 +335,7 @@ def _normalization(self, method: str) -> None:
 
     def _log2_transform(self):
         self.mat = np.log2(self.mat)
+        self.mat = self.mat.replace([np.inf, -np.inf], np.nan)
```

Comment:
TODO: This step should not be necessary if all downstream functions can handle infs.